packages/ripple-address-codec/test/index.test.ts

```diff
@@ -1,11 +1,11 @@
-const {
+import {
   classicAddressToXAddress,
   xAddressToClassicAddress,
   isValidXAddress,
   encodeXAddress,
-} = require('./index')
+} from '../src'
 
-const testCases = [
+const testCases: Array<[string, false | number, string, string]> = [
```

Comment:
Can you make an interface for this type to explain what these types mean? (It seems they may be `classicAddress`, `tag` | false, `mainnetAddress`, `testnetAddress` judging from the code on line 147)